ts/components/CompositionArea.tsx

```diff
@@ -43,7 +43,7 @@ export type Props = Pick<
   | 'onEditorSizeChange'
   | 'onEditorStateChange'
   | 'onTextTooLong'
-  | 'startingText'
+  | 'startingText' | 'clearQuotedMessage' | 'getQuotedMessage'
```

Comment:
there is a weird thing with linter failing on unrelated lines (71 and 73) if I split this into separate lines (however if I split this and remove 2 empty lines somewhere else it passes successfully - looks like a bug)